batchflow/models/torch/modules.py

```diff
@@ -0,0 +1,228 @@
+        num_features = np.prod(spatial)
+
+        phi = self.mid_branch(x).view(batch_size, -1, num_features) # (B, C/8, N)
+        theta = self.bot_branch(x).view(batch_size, num_features, -1) # (B, N, C)
+        attention = torch.bmm(phi, theta) / num_features # (B, C/8, C)
+
+        out = self.top_branch(x).view(batch_size, num_features, -1) # (B, N, C/8)
+        out = torch.bmm(out, attention).view(batch_size, -1, *spatial)
+        return self.gamma*out + x
+
+
+class FPA(nn.Module):
+    """ Feature Pyramid Attention.
+    Hanchao Li, Pengfei Xiong, Jie An, Lingxue Wang.
+    Pyramid Attention Network for Semantic Segmentation <https://arxiv.org/abs/1805.10180>'_"
+
+    Parameters
+    ----------
+    pyramid_kernel_size: list of ints
+        Kernel sizes in pyramid block convolutions
+    layout: str
+        Layout for convolution layers.
+    downsample_layout: str
+        Layout for downsampling layers. Default is 'p'
+    upsample_layout: str
+        Layout for upsampling layers. Default is 't
+    factor: int
+        Scaling factor for upsampling layers. Default is 2
+    bottleneck : bool, int
+        If True, then add a 1x1 convolutions before and after pyramid block with that factor of filters reduction.
+        If False, then bottleneck is not used. Default is False.
+    use_dilation: bool
+        If True, then the convolutions with bigger kernels in the pyramid block are replaced by top one
+        with corresponding dilation_rate, i.e. 5x5 -> 3x3 with dilation=2, 7x7 -> 3x3 with dilation=3.
+        If False, the dilated convolutions are not used. Default is False.
+    """
+    def __init__(self, inputs=None, pyramid_kernel_size=(7, 5, 3), bottleneck=False, layout='cna',
+                 downsample_layout='p', upsample_layout='t', factor=2, use_dilation=False, **kwargs):
+        super().__init__()
+        depth = len(pyramid_kernel_size)
+        spatial_shape = get_shape(inputs)[2:]
+        num_dims = get_num_dims(inputs)
+        self.attention = ConvBlock(inputs=inputs, layout='V >' + layout + 'b', kernel_size=1,
+                                   filters='same', shape=spatial_shape, dim=num_dims, **kwargs)
+
+        enc_layout = ('B' + downsample_layout + layout) * depth # B pcna B pcna B pcna
+        emb_layout = layout + upsample_layout # cnat
+        combine_layout = ('+' + upsample_layout) * (depth - 1) + '*' # +t+t*
+        main_layout = enc_layout + emb_layout + combine_layout # B pcna B pcna B pcna cnat +t+t*
+        main_strides = [1] * (depth + 1) + [factor] * depth # [1, 1, 1, 1, 2, 2, 2]
+
+        # list with args for BaseBlocks of each branch
+        branches = [dict(layout=layout, filters='same', kernel_size=1)] # the mid branch from inputs tensor directly
+
+        if use_dilation:
+            base_kernel_size = pyramid_kernel_size[-1] # 3
+            main_kernel_size = [base_kernel_size] * (depth + 1) + [factor] * (depth) # 3 3 3 3 2 2 2
```

Comment:
If you want to use some parameter for all transposed convolutions (`kernel_size = 3` in your case), you can use `transposed_conv={'kernel_size': 3}` once